### ISSUE TITLE
[AI] Support for Calculating carbon intensity

### ISSUE THREAD

**Issue by seanmcilroy29** [Green-Software-Foundation/software_carbon_intensity#25]
See Issue #22 
@buchananwp - To bring someone in from Microsoft with more formal experience in carbon measurement methodologies.

**Comment by buchananwp** [comment]
in-progress

**Comment by atg-abhishek** [comment]
We'll have more news on this over the next week as we reach out to some folks within Microsoft. If there are others in the broader community that we could tap into / speak with / invite, please feel free to share suggestions folks!

**Comment by buchananwp** [comment]
Propose we retitle this as 'Carbon Methodology Support'? 

@atg-abhishek and I have two folks onboarding now :)

**Comment by atg-abhishek** [comment]
Yes @buchananwp that sounds good, please feel free to do so and we can track other external expertise requests here as well. Thanks!
